public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -15,9 +15,10 @@ export interface Props {
   onChange: (update: LokiQuery) => void;
   onRunQuery: () => void;
   app?: CoreApp;
+  dsMaxLines: number;
```

Comment:
Just a nit here, but I think we prefer to list the optional properties in an object last.